src/lint/collect-spelling-diagnostics.ts

```diff
@@ -42,7 +42,8 @@ const matchers = [
   },
   {
     // this needs its own rule to catch +0 as a real number
-    pattern: /(?<= )\+[0-9]/gu,
+    // (but not similar text such as expanded-year dates like +000000-01-01)
+    pattern: /(?<= )\+[0-9](?![0-9]*[+-])/gu,
```

Comment:
What is the `+` for here?

```suggestion
    pattern: /(?<= )\+[0-9](?![0-9]*-)/gu,
```

Reply:
Hmm, I think I intended for that to be a `:` to prevent this rule from rejecting isolated UTC offsets like "+00:00".

Reply:
#463